scripts/deploy.sh

```diff
@@ -54,10 +54,11 @@ fi
 
 # PUBLISH: whether to publish/promote the deployed version
 PUBLISH="false"
-if [ "$BRANCH" = "master" ] && [ "$IS_PULL_REQUEST" = "false" ]; then
+if [ "$BRANCH" = "master" ] && ! [ "$IS_PULL_REQUEST" == "false" ]; then
```

Comment:
Should we also update the first portion of this conditional to use `==` as well?